UPGRADE.md

```diff
@@ -18,4 +18,8 @@ when that change will happen.
 
 1. The `pauseControllers` field in `master-config.yaml` is deprecated as of Origin 1.0.4 and will
   no longer be supported in Origin 1.1. After that, a warning will be printed on startup if it
-  is set to true.
\ No newline at end of file
+  is set to true.
+
+1. Currently, a SubjectAccessReview can be POSTed to either `/subjectaccessreview` or `/ns/namespace-name/subjectaccessreview`.  These endpoints have been superceded by 
```

Comment:
Lead with "The `/ns/namespace-name/subjectaccessreview` endpoint is deprecated ... use `/ns/namespace-name/localsubjectaccessreview` instead."